backend/src/database/seeds/999_production_seeder.js

```diff
@@ -23,6 +23,9 @@ exports.seed = (knex) => {
           const tableName = datFile.next();
           const columns = JSON.parse(datFile.next());
 
+          // eslint-disable-next-line no-console
+          console.log(`Loading table: ${tableName}`);
```

Comment:
I'm assuming we are leaving this in, as it gives some indication of the tables, as there are a lot of tables from production seeds.